Inc/driver.h

```diff
@@ -202,9 +204,16 @@
 #define SERIAL2_MOD
 #endif
 
+#ifndef SPI_PORT
+#define SPI_PORT 1
+#endif
+
 #if EEPROM_ENABLE|| KEYPAD_ENABLE || (TRINAMIC_ENABLE && TRINAMIC_I2C)
   #if defined(NUCLEO_F411) || defined(NUCLEO_F446)
     #define I2C_PORT 1 // GPIOB, SCL_PIN = 8, SDA_PIN = 9
+  #elif defined(BOARD_SKR_PRO_1_1)
+    #define I2C1_ALT_PINMAP
+    #define I2C_PORT 1 // GPIOB, SCL_PIN = 6, SDA_PIN = 7
```

Comment:
IMO these lines should be moved to the SKR board map and the code in this file changed to allow that.

Reply:
OK, I made that change